src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -177,7 +179,8 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
             FunctionContext? context = null;
             InvocationResponse response = new()
             {
-                InvocationId = request.InvocationId
+                InvocationId = request.InvocationId,
+                Result       = new StatusResult()
```

Comment:
Should we do this assignment here? I think the original idea to not do here was to avoid an extra allocation if the invocation causes an exception being thrown. In that case, this `StatusResult` instance won't be used as the catch block is creating a new instance.

Reply:
Ah whoops, meant to _not_ new up a new StatusResult in the catch but modify